src/components/Header/index.tsx

```diff
@@ -1,23 +1,19 @@
 import { Link } from 'gatsby';
-import React, { KeyboardEvent } from 'react';
+import React, { MouseEvent } from 'react';
+import { ThemeToggler } from 'gatsby-plugin-dark-mode';
 import logoLight from '../../images/logos/nodejs-logo-light-mode.svg';
 import logoDark from '../../images/logos/nodejs-logo-dark-mode.svg';
-import defaultDarkModeController from '../../util/darkModeController';
 import { useMediaQuery } from '../../hooks/useMediaQuery';
 
-interface Props {
-  darkModeController?: typeof defaultDarkModeController;
-}
-
-const Header = ({
-  darkModeController = defaultDarkModeController,
-}: Props): JSX.Element => {
+const Header = (): JSX.Element => {
   const isMobile = useMediaQuery('(max-width: 870px)');
 
-  const keyPressDarkModeHandler: (
-    e: KeyboardEvent<HTMLButtonElement>
-  ) => void = e => {
-    if (e.charCode === 13 || e.charCode === 32) darkModeController.toggle();
+  const handleThemeOnClick = (
+    e: MouseEvent<HTMLButtonElement, Event>,
+    toggleTheme: Function
```

Comment:
Adding `// eslint-disable-next-line @typescript-eslint/ban-types` should disable your linting error here and on line 91.

Reply:
thanks!